test-data/ffindex_indices.loc

```diff
@@ -0,0 +1,5 @@
+##ffindex indices
+#unique_id	display name	path	type
```

Comment:
This is one example config file that we use for testing ... you see here how I reference your indices.